src/components/Search/Input.js

```diff
@@ -45,20 +75,40 @@ const Input = ({ query, setQuery, refine, ...rest }) => {
     event.preventDefault()
   }
 
-  const intl = useIntl()
-  const searchString = translateMessageId("search", intl)
+  const focusSearch = (event) => {
+    const searchInput = document.getElementById("header-search")
+    if (document.activeElement !== searchInput) {
+      event.preventDefault()
+      searchInput.focus()
+    }
+  }
+  useKeyPress("/", focusSearch)
+
+  useEffect(() => {
+    setShowSearchSlash(window.innerWidth >= 768)
+  }, [])
```

Comment:
This should break at our themed breakpoint of "1024px" (when mobile view starts)... ideally using the theme context, ie:

First import ThemeContext: `import { ThemeContext } from "styled-components"`
Then something like this:
```
  const themeContext = useContext(ThemeContext)
  const breakpoint = themeContext.breakpoints.l
```